contracts/Governance.sol

```diff
@@ -88,12 +88,13 @@ contract Governance is TellorVars{
         address _oracle = IController(TELLOR_ADDRESS).addresses(_ORACLE_CONTRACT);
         require(IOracle(_oracle).getBlockNumberByTimestamp(_requestId, _timestamp) != 0, "Mined block is 0");
         address _reporter = IOracle(_oracle).getReporterByTimestamp(_requestId,_timestamp);
-        bytes32 _hash = keccak256(abi.encodePacked(_requestId, _timestamp));
+        bytes32 _hash = keccak256(abi.encodePacked(_requestId, _timestamp, _reporter));

```

Comment:
You don't want to hash the reporter, the dispute shouldn't matter who it's started from

Reply:
agreed